example_configs/dropwizard-0-8-x.yml

```diff
@@ -0,0 +1,153 @@
+---
+lowercaseOutputLabelNames: true
+lowercaseOutputName: true
+blackListObjectNames: ["metrics:type=*,name=jvm*"]
+
+rules:
+
+- pattern: "metrics<name=ch.qos.logback.core.Appender.(debug|error|info|trace|warn)><>Count"
+  name: log_msg_total
```

Comment:
These metrics are coming from logback, thus they should have 'logback' in the name. It's important to be able to guess where a metric is from from it's name.